modules/afsocket/afsocket-dest.c

```diff
@@ -207,6 +233,42 @@ afsocket_dd_start_reconnect_timer(AFSocketDestDriver *self)
   iv_timer_register(&self->reconnect_timer);
 }
 
+static gboolean
+_afsocket_dd_rename_connection_persist_name(AFSocketDestDriver *self, PersistState *state)
+{
+  const gchar *old_name, *new_name;
+  PersistEntryHandle old_handle, new_handle;
+  gsize size;
+  guint8 old_version, new_version;
+  gpointer old_value, new_value;
```

Comment:
If my previous suggestion can't be done, and it's not worth extracting anything from this method, then please declare these variables where they initialized. At least this reduces the scope of these 9 variables and makes the function a bit shorter.

Reply:
Thank You for the suggestion! I even extracted the actual move part to an inline function, so those variables are local now.

(I found extracting that part a little bit hard, because it will cause lots of output parameters in the extracted function.)

Reply:
Thanks.